platform/mac/menu.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="16097.2" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES" customObjectInstantitationMethod="direct">
+    <dependencies>
+        <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="16097.2"/>
+    </dependencies>
+    <objects>
+        <customObject id="-2" userLabel="File's Owner" customClass="NSApplication"/>
+        <customObject id="-1" userLabel="First Responder" customClass="FirstResponder"/>
+        <customObject id="-3" userLabel="Application" customClass="NSObject"/>
+        <menu title="Main Menu" systemMenu="main" id="AYu-sK-qS6">
+            <items>
+                <menuItem title="KOReader" id="1Xt-HY-uBw">
+                    <modifierMask key="keyEquivalentModifierMask"/>
+                    <menu key="submenu" title="KOReader" systemMenu="apple" id="uQy-DD-JDr">
+                        <items>
+                            <menuItem title="About KOReader" id="5kV-Vb-QxS">
+                                <modifierMask key="keyEquivalentModifierMask"/>
+                                <connections>
+                                    <action selector="orderFrontStandardAboutPanel:" target="-1" id="Exp-CZ-Vem"/>
```

Comment:
Are those `id`s purposefully random?

Reply:
I created the main menu in XCode -> New -> File -> MainMenu and removed most of options that are useless to us. Then renamed `MainMenu.xib` to `menu.xml` and here is the result.

As the Interface Builder stuff isn't linked to any code by itself it should work even if we modify their id to some other value. It should be still forwarded to the first responder, which is the SDL application.

Reply:
changed to simpler ids: 1 .. n